### DISCUSSION TITLE
Center system tray icons on vertical taskbar

### DISCUSSION THREAD

**Discussion by Zombiefied7** [valinet/ExplorerPatcher#688] [Ideas]
I use this project to move the taskbar to the left side in windows 11. Thanks for saving me.

When the taskbar is located on the left side of the screen, the icons in the system tray are not centered. Instead they are a little bit off to the right. This is a windows problem and been bugging me forever. Maybe you can fix it? Thanks for considering

**Comment by Zombiefied7** [comment]
![uncenteredSystemTray](https://user-images.githubusercontent.com/35601299/149840116-33a682c2-6aa9-4dc4-952c-fc334ecd32cd.PNG)

**Reply by valinet** [reply]
You can try making the taskbar a touch larger in order to fit 3 columns of icons: right click the taskbar, uncheck "Lock all taskbars", drag edge to resize.

It's pretty tricky to get code for this working right, let alone finding out where it is supposed to go. If I figure something out at some point, I will add it in, for sure.

Thanks.

**Comment by lapo-luchini** [comment]
This is what happens to me on minimum size (left) and just a few pixels more (right):
![image](https://user-images.githubusercontent.com/420454/152822935-d58fc1b9-a70d-48e9-b739-c66e98ba99d1.png)
What I'd like is to have two columns (or three) horizontally centered columns, this is what I currently have on Windows 10 (on a different PC):
![image](https://user-images.githubusercontent.com/420454/152823379-81e82c0c-655d-42ef-a512-8cadc3fcc601.png)

**Reply by lapo-luchini** [reply]
If horizontal centering is a problem, I could be already quite happy if the space between the icons was much less, vertical alignment can be "manually adjusted" with width of the overall bar.
